models/mesh.go

```diff
@@ -120,8 +125,34 @@ type ControlPlaneConfiguration struct {
 }
 
 type Certificate struct {
-	DNSNames   []string `yaml:"dnsNames"`
-	SecretName string   `yaml:"secretName"`
+	DNSNames      []string  `yaml:"dnsNames"`
+	ConfigMapName string    `json:"configMapName"`
+	Issuer        string    `json:"issuer"`
+	NotBefore     time.Time `json:"notBefore"`
+	NotAfter      time.Time `json:"notAfter"`
+	Error         string    `json:"error"`
+	Accessible    bool      `json:"accessible"`
+	ClusterName   string    `json:"cluster"`
```

Comment:
I noticed the existing settings have "yaml" in the struct tags. However, this new code uses "json". Seems odd that sometimes we have yaml and sometimes its json - we should be consistent and use the ones we need (or specify both).

Reply:
Well, I think the right one is json in this case, I fixed one tag what was still using yaml.